### ISSUE TITLE
Sección de agradecimientos.

### ISSUE THREAD

**Issue by marianagh** [CodeandoMexico/terremoto-cdmx#137]
Hace falta la sección para los agradecimientos se propone el siguiente [diseno](https://www.figma.com/file/WWr0IfDxJLIn26YuOYOTuQlz/cmx_sismomx_website_01)

**Comment by JPIglesias-Triangle** [comment]
Podríamos ponerlos como un modal?

**Comment by agush22** [comment]
1. Ya hay alguna libreria de modal en la pagina?
2. De donde se saca la lista de nombres?

**Comment by marianagh** [comment]
No estoy al tanto de eso me pidieron crear el issue con el diseño de referencia.

**Comment by agush22** [comment]
Yo hago este ☝️

**Comment by agush22** [comment]
Falta la lista de gente. 
Cuando la tengan me la pasan @gabelula

**Comment by gabelula** [comment]
Creo que iba a ponerse la gente que contribuyo al repo y al canal por lo que me parece. @tlacoyodefrijol  ?

**Comment by almarngl** [comment]
@gabelula @agush22 sí, se va a poner la lista de gente en el Slack + GitHub + empresas que han apoyado, en cuanto pueda les compilo la lista

**Comment by JPIglesias-Triangle** [comment]
Perfecto. Yo puedo trabajar en transcribirla.
